docs/source/pillars/information_governance.md

```diff
@@ -91,6 +91,14 @@ _The ability of the TRE organisation to monitor compliance with internal and ext
 * - You should report on and share outcomes of each audit of your TRE organisation with the required bodies.
   - This may be a requirement of continued accreditation by external organisations or regulatory bodies.
   - Recommended
+* - You log data access queries and publish these logs
+  - Transparency is a key principle for an environment that is trusted for research.
+    With these logs publicly available, there is transparency around how the data was used, and any actions that could compromise the security of the sensitive data.
+  - Recommended
+* - You log user actions other than data access, for example network connection attempts or infrastructure actions within a TRE.
+  - Logging can help to identify actions, intentional or unintentional, that could compromise the security of the SRE, or result in the accidental disclosure of data outside the TRE.
```

Comment:
I think I'm hesitant to say something like "logging identifies problems" because it can give a false sense of security. You could say it enables you to check for problems or they can be used to look for problems. The logs on their own won't do anything, but they can be inspected or you could set up alerts.

We also don't mention the value of logs in investigations _after_ an incident.

Reply:
Atm it says logging helps with this stuff rather than resolves it, do you think it should be clearer?

Reply:
Logs aren't useful in finding problems, you need an active monitoring tool for that. The tool might *use* log files, but the existence of logs without a monitoring tool is only useful for after-the-fact investigations, as Jim mentions.

Writing a tool like this is *hard*. You'd almost certainly be relying on a third-party solution (eg. Azure Monitor, Graphana etc.) and even configuring a third-party solution to be useful is hard.

Reply:
It sounds like we're in danger of going into too much detail- logging should be a routine part of IT operations for any organisation so can we refer to another standard? Is there anything _specific_ to TREs we should highlight?

Reply:
@manics Yes quite possibly.

I suppose what we need to focus on is,

- What aspects of logging are specific to TREs and,
- what aspects of logging are required to be considered a TRE.

I don't think this is what you were suggesting but trusting that logging is so natural that we don't need to mention it seems like a bad idea. Leaning on another standard or at least pointing to some good guidance might be a good way to do this. (We already mention, for example, ISO27001 elsewhere).